packages/pyright-internal/src/localization/package.nls.en-us.json

```diff
@@ -368,6 +368,7 @@
         "starPatternInOrPattern": "Star pattern cannot be ORed within other patterns",
         "starStarWildcardNotAllowed": "** cannot be used with wildcard \"_\"",
         "staticClsSelfParam": "Static methods should not take a \"self\" or \"cls\" parameter",
+        "stdlibModuleOverridden": "\"{path}\" is overriding the stdlib module \"{name}\". Try renaming \"{path}\" to something else.",
```

Comment:
Diagnostic messages don't end in periods and are never multiple sentences. Options to fix this are: 1) separate clauses with a semicolon, 2) move second sentence to a diagnostic addendum so it appears on a second line, or 3) drop the second sentence entirely. "Try renaming {path} to something else" is also redundant and overly verbose. Should be "Rename {path}". (In the immortal words of Yoda, "Do or do not. There is no try.") :)

Reply:
Thanks for the feedback. I think I'll remove the second sentence, especially since the quick fix action suggess a rename.